src/DotVVM.Framework/Controls/UpdateProgress.cs

```diff
@@ -93,7 +93,7 @@ public static IEnumerable<ControlUsageError> ValidateUsage(ResolvedControl contr
             {
                 if ((int)delayProperty.Value < 0)
                 {
-                    yield return new ControlUsageError("Delay cannot be set to negative number.");
+                    yield return new ControlUsageError("BackgroundCompilationDelay cannot be set to negative number.");
```

Comment:
No, this is wrong :sweat_smile:

Reply:
Ups :O
I've reverted the change and modified all the messages in UpdateProgress to use nameof